articles/dashboard/guides/tenants/configure-device-user-code-settings.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Configure Device User Code Settings
+description: Learn how to configure the user code generated by your applications during the device authorization flow using the Auth0 Management Dashboard.
+topics:
+  - device-auth
+  - api-authentication
+  - device-flow
+  - native-apps
+  - desktop-apps
+  - mobile-apps
+  - devices
+  - dashboard
+contentType:
+  - how-to
+useCase:
+  - secure-api
+  - call-api
+---
+# Configure Device User Code Settings
+
+Go into your tenant and navigate to `/tenant/advanced` and scroll down to *Device Flow User Code Format* section. You can configure the character set and the length.
```

Comment:
Is this supposed to be the first sentence? It seemed out of place, especially in relation to the next sentence

Reply:
Whoops! Good catch. No, it was text I had copied from Slack and was using to walk through the steps while writing. Removed. Sorry about that!